modules/transactions/src/main/java/org/apache/ignite/internal/tx/DeadlockPreventionPolicy.java

```diff
@@ -27,6 +27,11 @@
  * See also {@link org.apache.ignite.internal.tx.impl.HeapLockManager}.
  */
 public interface DeadlockPreventionPolicy {
+    /**
+     * No-op policy which does nothing to prevent deadlocks.
+     */
+    DeadlockPreventionPolicy NO_OP = new DeadlockPreventionPolicy() {};
```

Comment:
Minor. Because we have NO_OP explicitly we may use it in `NoneDeadlockPreventionTest`.